postgresw/pom.xml

```diff
@@ -70,6 +71,11 @@
             <classifier>shaded</classifier>
          <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
I don't think we need a new dependency. If all you need is parsing a string into json, you could use `com.arcadedb.serializer.json.JSONObject` or even `com.google.gson.JsonObject` that are already included

Reply:
fixed in 6fe20af05bc755a62deaebaa836d8c32de718579